"... is not recognized as an internal or external command"

looks like trouble in quotes, [concurrently](https://github.com/kimmobrunfeldt/concurrently) works fine when I run `concurrent --kill-others "npm run mocha" "selenium-standalone start" "node bin/api.js"`

> windows 10x64
> better-npm-run@0.0.5
> node v4.2.2
> npm  v3.5.2

```sh
>npm run test

> ...@9.0.0 test 
> better-npm-run test

running better-npm-run in ...
Executing script: test

to be executed: concurrent --kill-others "npm run mocha" "selenium-standalone start" "node bin/api.js"
[0] 'np' is not recognized as an internal or external command,
[0] operable program or batch file.
[1] 'run' is not recognized as an internal or external command,
[1] operable program or batch file.
[2] '"mocha\""' is not recognized as an internal or external command,
[2] operable program or batch file.
[3] 'selenium-standalon' is not recognized as an internal or external command,
[3] operable program or batch file.
[4] '"start\""' is not recognized as an internal or external command,
[4] operable program or batch file.
[5] 'nod' is not recognized as an internal or external command,
[5] operable program or batch file.
[6] '"bin/api.js\""' is not recognized as an internal or external command,
[6] operable program or batch file.
[6] bin/api.js" exited with code 1
[5] nod exited with code 1
[4] start" exited with code 1
[3] selenium-standalon exited with code 1
[2] mocha" exited with code 1
[1] run exited with code 1
[0] np exited with code 1

npm ERR! Windows_NT 10.0.10586
npm ERR! argv "E:\\Program Files\\nodejs\\node.exe" "C:\\Users\\Sigura\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "run" "test"
npm ERR! node v4.2.2
npm ERR! npm  v3.5.2
npm ERR! code ELIFECYCLE
npm ERR! ...@9.0.0 test: `better-npm-run test`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the ...@9.0.0 test script 'better-npm-run test'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the ... package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     better-npm-run test
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs ...
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls ...
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
```

package.json
```javascript
// ...
    "test": {
      "command": "concurrent --kill-others \"npm run mocha\" \"selenium-standalone start\" \"node bin/api.js\"",
      "env": {
        "FAKE_API": 1,
        "NODE_ENV": "production",
        "SERVER": "../test/.server",
        "TEST_BROWSER": "chrome",
        "RENDER_PATH": "../htdocs/bin/public"
      }
    },
// ...
```

--- a/lib/exec.js
+++ b/lib/exec.js
@@ -16,11 +16,13 @@ module.exports = function exec(script) {
 	if (process.platform === 'win32') {
 		sh = 'cmd';
 		shFlag = '/c';
+		command = '"' + command.trim() + '"';
 	}
 
 	console.log('to be executed:', command);
 	spawn(sh, [shFlag, command], {
 		env: env,
+		windowsVerbatimArguments: process.platform === 'win32',
 		stdio: ['pipe', process.stdout, process.stderr]
 	}).on('close', function(code) {
 		process.exit(code);

